src/main/java/Lesson3/Task1/ClassRoom.java

```diff
@@ -0,0 +1,23 @@
+package Lesson3.Task1;
+
+public class ClassRoom {
+    Pupil[] arrayPupil = new Pupil[4];
+
+    ClassRoom(Pupil ... pupil){
+        for(int i =0;i<pupil.length;i++){
+            arrayPupil[i] = pupil[i];
+        }
+    }
+
+    void print(){
```

Comment:
Think about better names for methods.
classRom.Print() doesn't give any meaningful info. It'd be better to use e.g. classRom.PrintAllPupulsInfo()